.github/ISSUE_TEMPLATE/feature-request-template.yml

```diff
@@ -0,0 +1,22 @@
+name: Feature Request
+description: File a feature request issue
+title: "[Feature Request]: <Please Replace This with a Summary of Your Request>"
+labels: ["addition", "p2", "needs triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you so much for taking the time to fill out this feature request report!
```

Comment:
optional: I think we can remove "report". Might just be a matter of opinion, but I tend to think of "bug report" and "feature request", not "feature request report".

Reply:
Makes sense. Done.